06-developer-software-guide.Rmd

```diff
@@ -11,46 +11,25 @@ The FIMS repository already has a file to set up all of the needed software, so
 
 To start a codespace on the FIMS repository, follow the [GitHub documentation for creating a Codespace](https://docs.github.com/en/codespaces/developing-in-a-codespace/creating-a-codespace-for-a-repository?tool=webui#creating-a-codespace-for-a-repository).
 
-## Software to Install
+## Software to install
 
 ### git
 
-You will need git installed locally, and you may prefer to use an
-additional git [GUI client](https://git-scm.com/downloads/guis) such as
-GitKraken or GitHub Desktop. If your preferred git client is the RStudio
-IDE, you can configure Git and RStudio integration following these
-[instructions](https://happygitwithr.com/rstudio-git-GitHub.html).
+To contribute to the code, you will need git installed locally, and you may prefer to use an additional git [GUI client](https://git-scm.com/downloads/guis) such as GitKraken or GitHub Desktop. See the section on [RStudio](#rstudio) for setting up RStudio to work with git.
```

Comment:
I would remove the GitKraken reference (licensing was a pain so nobody I know uses it any more) and change the final sentence to 
`See the sections on [RStudio](#rstudio) and [VS Code](#vs-code) for setting up those editors to work with git.`
Hyperlink may depend on subsequent section heading.

Reply:
Great suggestions